docroot/sites/all/themes/ilr_theme/template.php

```diff
@@ -143,7 +143,7 @@ function ilr_theme_preprocess_node(&$variables) {
       );
     }
     // See note above about hard-coded eq-pts
```

Comment:
@fjp2cornell the comment here says to see the note above, which will point you in the right direction. The link is in the docs for this function, ie `@see generate_event_item_markup() for hard-coded data-eq-pts`.